projects/plugins/jetpack/modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -1688,13 +1696,16 @@ protected function op_number_color($op, $lft, $rgt) {
 	protected function op_color_number($op, $lft, $rgt) {
 		if ($rgt[0] == '%') $rgt[1] /= 100;
 
+		$lftCount = is_countable( $lft ) ? count( $lft ) : 0;
```

Comment:
This seems like a case where more closely following the original semantics of `count()` would be better. Less likely to accidentally be passing -1 to `array_fill()`, which doesn't accept negative numbers.
```suggestion
		$lftCount = is_countable( $lft ) ? count( $lft ) : 1;
```
Although really I'd be surprised if this (or most of these really) ever is getting a non-array.